tensorflow_datasets/structured/heart_disease_test.py

```diff
@@ -0,0 +1,19 @@
+"""heart_disease dataset."""
```

Comment:
Add License Text at the start of the document.

Reply:
added